progress/uladzislau-muraveika.md

```diff
@@ -1,3 +1,14 @@
+**2024-07-19: Weekly update**
+
+My current week focus was the fixes for ongoing issues and small improvements.
+
+There were several topics from the infrastructure side. Automatic SSL renewal on our internal Grafana, benefit - we wouldn't be doing it manually. Removal of old in-house Explorer infra, benefit - illuminate double spending on infra costs. I've also applied rate-limits on the devnet and load tested how nginx will work under the load, advantage - mitigate load spikes on the RPC endpoints. And one item, after complaints from on-call engineers, I've added separate disk usage alarm with higher threshold for our archival and snapshot nodes, this reduced our monitoring noise.

```

Comment:
fix typos thanks

Reply:
fixed and made separate points,  please merge